tests/unittests/benchmark/task/test_tasks.py

```diff
@@ -101,3 +113,72 @@ def test_search_space(self):
         """Test to get task search space"""
         task = RosenBrock(2)
         assert task.get_search_space() == {"x": "uniform(-5, 10, shape=2)"}
+
+
+@pytest.mark.skipif(
+    IMPORT_ERROR is not None,
+    reason="Running without HPOBench",
+)
+class TestHPOBench:
+    """Test benchmark task HPOBenchWrapper"""
+
```

Comment:
This can be true if we have `singularity` installed.  But we can also test invalid benchmark names without `singularity`, because invalid name will cause the creation of benchmark fail before it reachs the point of using `singularity`.